core/src/main/scala/kafka/coordinator/group/GroupMetadata.scala

```diff
@@ -112,18 +116,32 @@ private object GroupMetadata {
       PreparingRebalance -> Set(Stable, CompletingRebalance, Empty),
       Empty -> Set(PreparingRebalance))
 
+  val DefaultCurrentStateTimestamp: Long = -1
+
   def loadGroup(groupId: String,
```

Comment:
This overload feels like overkill. Can we just have the caller pass the timestamp?